Code/GraphMol/Wrap/Conformer.cpp

```diff
@@ -58,44 +52,6 @@ PyObject *GetPos(const Conformer *conf) {
   return PyArray_Return(res);
 }
 
-void SetPos(Conformer *conf, np::ndarray const & array) {
```

Comment:
Looks like you accidentally reverted some changes

Reply:
oh I couldn't build, I'll drop that reversion